test/models/user_test.rb

```diff
@@ -0,0 +1,67 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+  def setup
+  	@user = User.new(name: "Example User", email: "user@example.com", password: "foobar", password_confirmation: "foobar")
```

Comment:
Line is too long. [121/80]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.